packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1348,6 +1356,9 @@ export class ContainerRuntime
 			"Fluid.ContainerRuntime.Test.CloseSummarizerDelayOverrideMs",
 		);
 		this.closeSummarizerDelayMs = closeSummarizerDelayOverride ?? defaultCloseSummarizerDelayMs;
+		this.validateSummaryBeforeUpload =
+			this.mc.config.getBoolean("Fluid.ContainerRuntime.Test.ValidateSummaryBeforeUpload") ??
+			true;
```

Comment:
You're switching to default false right?

Reply:
Yes, I will do that.

Reply:
done.